### DISCUSSION TITLE
動茲券都怎麼用

### DISCUSSION THREAD

**Discussion by tzurou** [1101-GS4719-C/Python-Community-GS4719-C#1982] [General]
去年政府發的動茲券到現在都還沒用，6月底就要過期了不想浪費，今年的也完全沒動到，已經最後一年領了，很想去大巨蛋看棒球，但很遠一直找不到有空的時間，感覺要揪到一群人一起去看棒球也不容易，想問大家都是怎麼用動茲券的，我知道動茲券是想讓我運動，但我對運動興趣缺缺，或許有一些旁門左道可以用嗎？

**Comment by YOGA012** [comment]
我都拿去國民運動中心買游泳池的入場券，要找那種有spa區的，如果不喜歡游泳，可以直接在裡面的spa區泡泡熱水、沖水按摩或到桑拿房出出汗，每當我想放鬆時就會到spa區舒服舒服。

**Comment by yxin214** [comment]
我都拿去大魯閣打保齡球和打棒球！如果願意自己貼一點錢，帶全家一起去打保齡球很不錯，平常沒在玩的話三、四局手就蠻酸的了，是增進全家感情的好去處。

**Comment by EPHMR0524** [comment]
我根本沒用，事情太多了根本沒有時間看展覽或是找些活動去使用動茲券，就是放著等他過期的，也不知道這個東西能不能轉讓給別人，畢竟我自己是用不到的，但對於平常有去健身或是去游泳館的人來說這應該是相當不錯的小補助。

**Comment by lilishh** [comment]
如果喜歡打棒球的話，推薦去大魯閣辦學生會員，辦學生會員的優惠是周一可以暢打某些球道！另外也會送很多代幣，個人覺得划算，所以已經拿動茲券辦兩次會員了。

**Comment by cassy0201** [comment]
我覺得想要娛樂性質的話可以選滑輪，若想要運動，游泳、健身房也是不錯的選擇

**Comment by poiu6fi** [comment]
之前的快到期了，暑假要趕快用一用，想去打羽球或桌球之類的

**Comment by YeeDaniel** [comment]
我個人有一個還不錯的想法，如果你真的沒有機會去看棒球比賽或是參加比賽的話，可以考慮轉賣給需要的人唷！因為有一些三鐵或是路跑比賽會開放使用動資券，然後...比賽畢竟都蠻貴的，所以用動資券蠻划算的~ 
(個人其實是有這個需求哈哈，不會用到的話可以跟我聯絡~)

**Comment by JustAHedgehog** [comment]
我是利用動滋券訂閱健身房的會員，選的是健身工廠，他們推出的優惠是使用動滋券將提供免費一個月的訂閱，距離離學校也不遠，目前打算是暑假花兩個月的時間每周去健身房一次，以此改善身體狀況

**Comment by JustAHedgehog** [comment]
我是利用動滋券訂閱健身房的會員，選的是健身工廠，他們推出的優惠是使用動滋券將提供免費一個月的訂閱，距離離學校也不遠，目前打算是暑假花兩個月的時間每周去健身房一次，以此改善身體狀況

**Comment by OG-Matcha** [comment]
我自己是比較少在這方面的花費，所以就不太會用到動滋券，他已經快從我的腦袋淡忘了，直到有一天我的同學和我說他想要低價收購我的動滋券，我細思了一下發現真香，所以你需要一個會用動滋券朋友。

**Comment by stone971275** [comment]
我是拿去看球賽，還有攀岩，有些還會有特別的優惠。

**Comment by ds0w0** [comment]
我是幾乎沒有在這方面的花費，所以不太會用到動滋券。之前我有朋友是去大魯閣辦學生會員，因為他很常去那邊打棒球，上次約我去他就當場在那邊辦。我們家最後是拿去買蛙鏡和泳帽，雖然不常玩水，但因為學校要用，家裡的也該換新的了就買了，其實蠻容易就花完了，如果要租什麼場地或買健身房會員應該也可以。

**Comment by JoannaLai909** [comment]
可以拿去買愛爾達會員看奧運，但我個人是都用來租借運動中心的球場或買運動用品

**Comment by peresonacifer** [comment]
因為我沒有運動的習慣，原本並不打算領動滋券，但我同學推薦我可以去運動用品店買衣服或襪子，所以如果你和我一樣喜歡宅在家，可以這麼做，不然就和同學們約出去玩，畢竟動滋券不花白不花。

**Comment by MingWei174** [comment]
平時對棒球沒有太多的接觸，在領到動滋券後本來以為毫無用途，但在同學的推薦下我把它拿去大魯閣棒壘球場開了兩道球道練習看看，沒想到愈打愈上手，對棒球也愈來愈有興趣，並希望未來如果又有動滋券的話可以繼續和同學去打棒球!

**Comment by hunteryang0108** [comment]
動茲券不一定要用在體育項目上，有一些比較輕鬆的遊戲項目也可以使用，例如：保齡球館、桌球館、電競館。我自己是使用在游泳館，我認為保持良好的運動習慣可以維持身體健康，相信這也是政府推行動茲券的目的吧。

**Comment by Jerryguo888** [comment]
除了運動外，動茲券也可以用在休閒與文化活動像是購買音樂會或演出的門票，或者如果有對音樂或舞台表演有興趣，可以考慮這種方式。另外還可以用於博物館、藝術展覽等文化活動。
